masterlist.yaml

```diff
@@ -2302,6 +2302,30 @@ plugins:
         udr: 4
 
 ###### Gameplay - Combat ######
+  - name: 'Forever Radiation Lite.esp'
+    url: [ 'https://www.nexusmods.com/fallout4/mods/48231/' ]
+    dirty:
+      - <<: *quickClean
+        crc: 0x4A9C4968
+        util: '[FO4Edit v4.0.4](https://www.nexusmods.com/fallout4/mods/2737)'
+        itm: 7
+    clean:
+      - crc: 0x004D1427
+        util: 'FO4Edit v4.0.4'
```

Comment:
We exclude cleaning data for dirty plugins after they're cleaned due to volatility in the checksum.

```suggestion
```